Contentful.ModelsCreator.Cli/ModelsCreator.cs

```diff
@@ -174,7 +175,8 @@ private string RemoveUnallowedCharacters(string s)
 
         private string GetSafeFilename(string filename)
         {
-            return string.Join("-", filename.Split(System.IO.Path.GetInvalidFileNameChars()));
+            var withoutInvalidPathChars = string.Concat(filename.Split(System.IO.Path.GetInvalidFileNameChars()));
+            return RemoveUnallowedCharacters(FirstLetterToUpperCase(withoutInvalidPathChars));
```

Comment:
This is the primary change. All the other edits are from automatic editor formatting. Let me know if you'd like me to remove the formatting changes to help readability.